GVFS.ruleset

```diff
@@ -0,0 +1,102 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Rules for GVFS" Description="Code analysis rules for GVFS projects" ToolsVersion="15.0">
+  <Rules AnalyzerId="Microsoft.Analyzers.ManagedCodeAnalysis" RuleNamespace="Microsoft.Rules.Managed">

```

Comment:
Is there a rule for "no var"? :-)

Reply:
I didn't see a StyleCop rule for this :)

However, I think var usage *is* actually controllable via [.editorconfig](https://docs.microsoft.com/en-us/visualstudio/ide/editorconfig-code-style-settings-reference?view=vs-2017#c-code-style-settings)...